sys/mips/Makefile

```diff
@@ -12,7 +12,7 @@ SOURCES = \
 	ebase.S \
 	gt64120.c \
 	interrupt.c \
-	$(BOARD).c \
+	malta.c \
```

Comment:
It's really strange thing you do in this commit. You add new variable `$ARCH`, but on the same time you want to remove usage of one - `BOARD`, but one the same time you don't remove declaration of that `BOARD` variable. 
Shouldn't you add more variable with their usage's or remove them by hardcoding more things?
BTW I don't think that hardcoding more things as you do it in this line is good.

Reply:
First of all `ARCH` already exists. `ARCH` is __current__ architecture. I have added `ARCHS` which is set of all supported architectures. It has been added because when you build mips (default) build system doesn't known about aarch64 subdirectory. As a result `make` doesn't call any recursion on that directory. Here I append every architecture for generic format target.
`BOARD` needs to be removed in that approach. Let's assume that you compile mips build. Then `BOARD=malta` and you call `sys/aarch64` with `BOARD=malta` and it's wrong because `malta.c` doesn't exist in `sys/aarch64`.

To be honest I would like to remove `BOARD`. We should build generic image for each architecture and initialize devices using information provided by dtb but it's not a target of that PR.